### ISSUE TITLE
Wrong sort order in mouseover tooltip for buy/sell in performance tab

### ISSUE THREAD

**Issue by siprbaum** [portfolio-performance/portfolio#1612]
Using the mouseover tooltip in the security performance tab (german: Wertpapier Performance) shows the individual orders unsorted, which might result in showing the sale before the actual buy.

The expectation would be to show them sorted by date.
Example of the current behaviour (image cropped to anonymize the actual amount):

![image](https://user-images.githubusercontent.com/18378462/85364893-76837c00-b524-11ea-8daf-02bbad3d816b.png)

Instead, it should show
- 07.09.2017 Kauf
- 02.10.2017 Verkauf
- 15.12.2017 Kauf
- 24.01.2019 Verkauf

Seen in Version:
`0.46.5 (Juni 2020) Platform: win32, x86_64 Java: 11.0.4+11-LTS, Azul Systems, Inc.`

**Comment by buchen** [comment]
Fair point.

The thinking was:
The tooltip shows the realized capital gains (realisierte Kurserfolge). And that is always

**Comment by siprbaum** [comment]
Interesting. I didn't look at it from that perspective, especially as the popup is not listing it like a mathematical sum (or difference) operation, e.g. like

```
  valuation at end (sale)
- valuation at start (sale) 
----------------------------
  value of the sum (or difference)
```
which would make the current existing sorting more natural.

With the left / right shift of the values my first association was that it is done like it is usually done in accounting (e.g. "Soll / Haben Seite") although I don't know much about about that.
```
  valuation at end (sale)
                              valuation at start (sale)                             
```

Furthermore, wanted to look at the individual transactions and was at first very glad that the popup provided that to me, but then I became highly confused about the sorting order and didn't even consider that the current sort order was done by intention.

So long story short, if you ask me directly, I would prefer the trades sorted by date!

**Comment by Sn1kk3r5** [comment]
it's been sorted now
